docker-compose.yaml

```diff
@@ -95,21 +96,36 @@ services:
   ingest:
     container_name: ingest
     image: stellar/wallet-backend:development
+    healthcheck:
+      test: "curl --fail --silent --show-error --location 'http://localhost:8002/health' | grep -q '\"status\": \"ok\"'"
+      interval: 10s
+      timeout: 10s
+      retries: 3
     build:
       context: ./
       dockerfile: Dockerfile
     depends_on:
       db:
         condition: service_healthy
-      api:
-        condition: service_healthy
       stellar-rpc:
         condition: service_started
     entrypoint: ""
     command:
       - sh
       - -c
-      - ./wallet-backend ingest
+      - |
+        ./wallet-backend migrate up
+        if [ "$STELLAR_ENVIRONMENT" = "GITHUB_WORKFLOW" ]; then
+          HEALTH_RESPONSE=$(curl -s -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1,"method":"getHealth"}' "${RPC_URL}")
+          LATEST_LEDGER=$(echo "$$HEALTH_RESPONSE" | grep -oE '"latestLedger":[0-9]+' | grep -oE '[0-9]+' || true)
+          if [ -z "$$LATEST_LEDGER" ] || [ "$$LATEST_LEDGER" = "" ]; then
+            ./wallet-backend ingest
+          else
+            ./wallet-backend ingest --start-ledger "$$LATEST_LEDGER"
+          fi
+        else
+          ./wallet-backend ingest
+        fi
```

Comment:
Why is this necessary? Shouldn't `ingest` check RPC itself rather than require the latest ledger to be provided via its CLI?

Reply:
As mentioned in the previous comment, since on fresh start, the ingest starts from the oldest ledger in the connected RPC instance, for the integration tests it starts from the oldest ledger in our testnet RPC and it takes forever for the pods to get healthy. So I specifically tell it to start from the latest ledger instead